src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -91,6 +92,28 @@ static void symbolValidator(ModuleSymbol module)
             }
         }
 
+        [Fact]
+        public void OutParameters()
+        {
+            var comp = CompileAndVerifyFunctionPointers(@"
+class C
+{
+    public unsafe void M(delegate*<out C, out string, ref int[]> param1) => throw null;
```

Comment:
>ref [](start = 54, length = 3)

Do we have a test for an attempt to use ```out``` on a return type?

Reply:
Added to `FunctionPointerTypeSymbolTests.InvalidReturnModifiers`

---
In reply to: [409814570](https://github.com/dotnet/roslyn/pull/43355#discussion_r409814570) [](ancestors = 409814570)